### PR TITLE
Speed up data queries for initial cellar data

### PR DESCRIPTION
**Pull request by philipjames44**
Speeds up the queries for the cellar overviews.

### CODE REVIEW

File: src/queries/get-all-strategies-data.graphql [hunk 1]
```diff
@@ -1,7 +1,20 @@
-query GetAllStrategiesData {
+query GetAllStrategiesData($monthoAgoEpoch: Int!) {
```
Comment (sandbochs) [inline src/queries/get-all-strategies-data.graphql:1]: is anything else using this query? should we make this change a new file?

**Review by sandbochs — APPROVED** [review]
some minors

**Comment by sandbochs** [comment]
Note for next full resync:

Put cellar_id field on the day data entity so we can sort by it.
Put 1d, 7d, and 30d share value change into cellar and day data entities.

File: src/queries/get-all-strategies-data.graphql (hunk 1 above)
Comment (philipjames44) [reply src/queries/get-all-strategies-data.graphql]: not as far as i can tell

File: src/data/actions/common/getAllStrategiesData.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export const getAllStrategiesData = async ({
         const result = await reactQueryClient.fetchQuery(
           ["USE_STRATEGY_DATA", { provider: true, address }],
           async () => {
-            const subgraphData = sgData.cellars.find(
+            let subgraphData = sgData.cellars.find(
```
[the remark this answers is not in the document]
Comment (philipjames44) [reply src/data/actions/common/getAllStrategiesData.ts]: fixed

**Pull request merged**